packages/ember-views/lib/components/component.js

```diff
@@ -351,6 +351,55 @@ var Component = View.extend(TargetActionSupport, {
     @property hasBlockParams
     @returns Boolean
   */
+
+  /**
+    Enables components to take a list of parameters as arguments
+
+    For example a component that takes two parameters with the names
+    `name` and `age`:
+
+    ```javascript
+    Ember.Component.extend({
+      positionalParams: ['name', 'age']
```

Comment:
The syntax slightly changed late in the 1.13 cycle. This syntax will result in a deprecation. 

It should be:

    let Component = Ember.Component.extend;
    Component.reopenClass({
      positionalParams: ['name', 'age']
    });